frontend/state/stores/runStore.js

```diff
@@ -28,17 +35,17 @@ class RunStore {
         this.runs[run.id] = new Run(run, socket);
     }
 
-    @action
+    @computed
```

Comment:
As far as I know, `@computed` functions are generally getters. This method should not need to use any decorator because it is neither an action nor a getter.

Reply:
I have fixed this